js/ui/slide_out_view.js

```diff
@@ -82,15 +82,17 @@ var SlideOutView = Widget.inherit({
             /**
             * @name dxSlideOutViewOptions_menuTemplate
             * @publicName menuTemplate
-            * @type template
+            * @type_function_param1 menuElement:dxElement
+            * @type template|function
```

Comment:
The "template" type already includes "function":
https://js.devexpress.com/Documentation/ApiReference/Common/Object_Structures/template/

Reply:
We need this to generate the `Function parameters:` block like [there](https://js.devexpress.com/Documentation/ApiReference/UI_Widgets/dxCalendar/Configuration/#cellTemplate)